tempesta_fw/cfg.c

```diff
@@ -1315,7 +1322,8 @@ tfw_cfg_set_str(TfwCfgSpec *cs, TfwCfgEntry *e)
 
 		min = cse->len_range.min;
 		max = cse->len_range.max;
-		if (min != max && (len < min || len > max)) {
+		if (((min != 0) && (max != 0)) &&
+			(len < min || len > max)) {
```

Comment:
Yes, you're right, i must have missed that comment. I looked though tempesta configuration once again and exact string length is not one of our use case. I will remove that code modifications and related unit test.